pym/portage/util/writeable_check.py

```diff
@@ -58,7 +58,11 @@ def linux_ro_checker(dir_list):
 				# to the left of the ' - ', after the attr's, so split it there
 				mount = line.split(' - ', 1)
 				_dir, attr1 = mount[0].split()[4:6]
```

Comment:
@zmedico yes but is there situation when it's possible and what should be taken if it's not [4:6]